src/api/article/controller.js

```diff
@@ -58,7 +65,7 @@ export const create = async ({ body }, res, next) => {
     const articleBrand = (await articleBrandQuery.exec()) || new ArticleBrand({ name: body.brand });
     await articleBrand.save();
 
-    const articleCollection = await ArticleCollection.findOne({ slug: body.collectionSlug });
+    const articleCollection = await ArticleCollection.findOne({ slug: body.collectionSlug }).exec();
```

Comment:
Doesn't this work without `exec`? What is difference?

Reply:
It works without `exec`. The difference is without exec it's a Mongoose Query (not a Promise) and after we call an exec() it's a real Promise. What do we need to await it then?

Reply:
ok, lets use `exec`